pkg/kev/kev.go

```diff
@@ -123,3 +124,72 @@ func Render(format string, singleFile bool, dir string, envs []string) error {
 
 	return nil
 }
+
+// Dev continuously watches source compose files and rebuilds Kubernetes manifests on each change.
+// Note: It'll rebuild manifessts for all tracked environments, unless list of environments specifically provided.
```

Comment:
Based on the implementation so far, `kev.Dev` just notifies the caller of changes to the manifest's sources.

It's up to the client to react to those changes - which I think makes a ton of sense.

**How about we rename `kev.Dev` to `kev.Watch`** and not pass in the `envs` param?

That way any complications from dealing with `envs` are up to the caller to deal with.

Reply:
changed

Reply:
@marcinc after reviewing the latest changes, I think there was a bit of misunderstanding - prob because I mentioned `any complications from dealing with envs` ... my bad.

We should catch up quickly after stand up.